src/lib/viewers/image/ImageViewer.js

```diff
@@ -513,6 +514,15 @@ class ImageViewer extends ImageBaseViewer {
      */
     handleAnnotationControlsClick({ mode }) {
         const nextMode = this.annotationControlsFSM.transition(AnnotationInput.CLICK, mode);
+
+        if (nextMode === AnnotationMode.REGION) {
+            if (!this.cache.get(IMAGE_FTUX_CURSOR_DISABLED_KEY)) {
+                this.cache.set(IMAGE_FTUX_CURSOR_DISABLED_KEY, true, true /* useLocalStorage */);
+            } else {
+                this.annotator.emit('annotations_image_explicit_create_toggled');
```

Comment:
It looks like we're both passing in options to the annotator and emitting events to it. Are they related? Can this be simplified?

Reply:
They are related:
- After a refresh of the page, we want to pass the ftux cursor flag via options to the annotator.
- When the user clicks on the Create Region button, we want to emit an event that would cause a rerender of the RegionCreator component.

I'm not aware of how this could be simplified but would be open to suggestions.

Reply:
We no longer need to pass in options to the annotator after implementing the css approach.